src/gcp_scanner/scanner.py

```diff
@@ -221,15 +222,27 @@ def crawl_loop(initial_sa_tuples: List[Tuple[str, Credentials, List[str]]],
       # Get storage buckets
       if is_set(scan_config, 'storage_buckets'):
         dump_file_names = None
+        dump_iam_file_names = None
         if scan_config is not None:
           obj = scan_config.get('storage_buckets', None)
           if obj is not None and obj.get('fetch_file_names', False) is True:
             dump_file_names = open(gcs_output_path, 'w', encoding='utf-8')
-        project_result['storage_buckets'] = crawl.get_bucket_names(project_id,
+          if obj is not None and obj.get('fetch_buckets_iam', False) is True: 
```

Comment:
Because I want to get buckets_iam_policy  in 1 JSON file with other info. Reason why I added this flag your comment: 

"We need to dump IAM policies together in a separate file similar to what we do above for storage_buckets. Sometimes the list is super huge and it really makes the main report hard to read." 

I сan't imagine that more than 10-15 users are attached to a bucket.

Reply:
Agree but you are also saving it in the main report at lines 425-427 or am I missing something?

Reply:
It's correct. My opinion that ok what you store all info in one file, but if dump_fd is not None, all permissions migrate to another files

Reply:
Ok, I got it. I think your original approach of pushing all of that into the main report just makes sense. No need for a separate file, sorry about confusion on my side. I provided full details in the comment below. Hope it works.